tests/test_fmi_coupled.py

```diff
@@ -15,81 +15,78 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-import pytest
 import os
+import pytest
 
 from pyfmi.fmi import FMUModelME2
 from pyfmi.fmi_coupled import CoupledFMUModelME2
 import pyfmi.fmi as fmi
-from pyfmi.tests.test_util import Dummy_FMUModelME2
+from pyfmi.test_util import Dummy_FMUModelME2
 
-assimulo_installed = True
 try:
     import assimulo
 except ImportError:
-    assimulo_installed = False
+    pass
 
 file_path = os.path.dirname(os.path.abspath(__file__))
-
 me2_xml_path = os.path.join(file_path, "files", "FMUs", "XML", "ME2.0")
 
-if assimulo_installed:
```

Comment:
I think this was a fundamentally flawed approach, if one doesn't have Assimulo installed, a user should not run the tests that require Assimulo instead.